### PR TITLE
add: added  and configured redux-toolkits

### PR DESCRIPTION
**Pull request by Djonatanabreu**
I've added the redux toolkit and configured
I've tested redux with register view

### CODE REVIEW

File: src/views/Register/index.tsx [hunk 1]
```diff
@@ -57,8 +74,9 @@ export const Register = () => {
     updateForm(fieldIndex, 'value', value);
   };
 
-  const handlerNextTep = () => {
-    console.log('CONTINUE');
+  const handlerNextStep = () => {
```
Comment (danielanthonyl) [inline src/views/Register/index.tsx:77]: suggestion: `handlerNextStep` should be `nextStepHandler`, `onNextStep` etc

Comment (Djonatanabreu) [reply src/views/Register/index.tsx]: fixed at the commit:

[04e9c66](https://github.com/AbreuLetitia/legendary-games/pull/12/commits/04e9c66cc3f662fd3166c77f2fb10850d07d596f)

File: src/views/Register/index.tsx [hunk 2]
```diff
@@ -33,6 +39,17 @@ export const Register = () => {
...
+    form.forEach((field) => {
+      userData = {
+        ...userData,
+        [field.id]: field.value,
+      };
+    });
+    return userData;
+  };
+
   const { navigate } = useNavigation();
```
[the remark this answers is not in the document]
Comment (Djonatanabreu) [reply src/views/Register/index.tsx]: fixed at the commit:

[04e9c66](https://github.com/AbreuLetitia/legendary-games/pull/12/commits/04e9c66cc3f662fd3166c77f2fb10850d07d596f)

File: src/store/userManage/types.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  name: string;
+  lastName: string;
+  displayName: string;
+  email: string;
+  password: string;
+}
+
+type UserActionCreator<payload> = ActionCreator<IUserState, payload>;
+
+export type IUserActions = {
```
[the remark this answers is not in the document]
Comment (Djonatanabreu) [reply src/store/userManage/types.ts]: fixed at the commit:

[04e9c66](https://github.com/AbreuLetitia/legendary-games/pull/12/commits/04e9c66cc3f662fd3166c77f2fb10850d07d596f)

File: src/store/store.ts [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import { configureStore } from '@reduxjs/toolkit';
+
+import userReducer from './userManage';
+
+export const store = configureStore({
+  reducer: {
+    registeredUsers: userReducer,
```
[the remark this answers is not in the document]
Comment (Djonatanabreu) [reply src/store/store.ts]: fixed at the commit:

[04e9c66](https://github.com/AbreuLetitia/legendary-games/pull/12/commits/04e9c66cc3f662fd3166c77f2fb10850d07d596f)

**Review by danielanthonyl — APPROVED** [review]

**Pull request merged**